docs/getting_started/faq.rst

```diff
@@ -0,0 +1,112 @@
+Frequently Asked Questions (FAQs)
+=================================
+
+Below, you will find answers to the most commonly asked questions about
+Modin. If you still cannot find the answer you are looking for, please post on 
+the #support channel on our Slack_ community or open a Github issue_.
+
+What’s the issue with pandas? Why should I use Modin?
+""""""""""""""""""""""""""""""""""""""""""""""""""""""
+While pandas works extremely well on small datasets, as soon as you start working with 
+medium to large datasets that are more than a few GBs, pandas can become painfully 
+slow or run out of memory. This is because pandas is single-threaded. In other words, 
+you can only process your data with one core at a time. This approach does not scale to 
+larger data sets and adding more hardware does not lead to more performance gain. 
+
+The ``modin.pandas`` `DataFrame`_ is a highly scalable, parallel DataFrame. Modin
+transparently distributes the data and computation so that you can
+continue using the same pandas API while being able to work with more data faster. With Modin, 
+you are able to use all of the CPU cores on your machine, and because of it's light-weight
+nature often results in less memory overhead than pandas. See this 
```

Comment:
The "With Modin" sentence seems awkward . I suggest taking out the "you" and simplifying the clause about memory:

```suggestion
continue using the same pandas API while being able to work with more data faster. Modin lets you use all the CPU cores on your machine, and because it is lightweight, it often has less memory overhead than pandas.
```

Reply:
LGTM!